src/compat/dune

```diff
@@ -0,0 +1,3 @@
+(library
```

Comment:
This library could be better defined in `src/odoc` to not suggest the library is public. Perhaps in a sub directory: `src/odoc/compat`